lib/puppet/provider/service/upstart.rb

```diff
@@ -1,5 +1,3 @@
-require 'semver'
```

Comment:
I did consider to keep a tiny class that just delegated everything to `SemanticPuppet` but decided against it. It is deprecated now and 5.0 is the new major release where its removal is scheduled to take place. I don't think an empty file would be any better. After all, if some module requires the presence of the `semver.rb` file, isn't it very likely that it will break anyway? Why require it if nothing in it is referenced?